src/org/javarosa/core/model/FormDef.java

```diff
@@ -1525,6 +1528,12 @@ private void attachControlsToInstanceData(TreeElement node) {
         }
     }
 
+    /**
+     * Returns the given question definition related to a form node reference.
+     * @param ref Reference to a form node.
+     * @param fe IFormElement, typically a FormDef, often 'this'.
```

Comment:
“often 'this'”? What is “this”?

Reply:
Corrected this one too. Marking resolved.